static/app/components/profiling/FrameStack/frameStackContextMenu.tsx

```diff
@@ -34,7 +35,13 @@ export function FrameStackContextMenu(props: FrameStackContextMenuProps) {
             {...props.contextMenu.getMenuItemProps()}
             onClick={props.onZoomIntoFrameClick}
           >
-            {t('Show in flamegraph')}
+            {t('Show on flamechart')}
+          </ProfilingContextMenuItem>
+          <ProfilingContextMenuItem
+            {...props.contextMenu.getMenuItemProps()}
+            onClick={props.onHighlightAllFramesClick}
+          >
+            {t('Highlight all occurrences')}
```

Comment:
The terminology here might need some clarification. It's not immediately obvious what it will high all occurrences of.

Reply:
How about `Highlight all frame occurrences`? We cant go too long or that context menu will take half the screen :D